mailpoet/lib/Automation/Integrations/MailPoet/Templates/Templates.php

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types=1);
+
+namespace MailPoet\Automation\Integrations\MailPoet\Templates;
+
+use MailPoet\Automation\Engine\Workflows\Step;
+use MailPoet\Automation\Engine\Workflows\Workflow;
+use MailPoet\Automation\Integrations\Core\Actions\WaitAction;
+use MailPoet\Automation\Integrations\MailPoet\Actions\SendWelcomeEmailAction;
+use MailPoet\Automation\Integrations\MailPoet\Triggers\SegmentSubscribedTrigger;
+
+class Templates {
```

Comment:
Please tell me if you can think of a better place to put this logic. I wanted it to exist somewhere outside the controllers though because it feels like something we may want to use outside the context of API requests, and it's more straightforward to test this way.

Reply:
Putting it outside the controllers sounds good to me. I'm thinking if we can improve the naming.

The class basically generates a Workflow (with step IDs and stuff) so maybe it could be `...\Templates\WorkflowGenerator` or `WorkflowBuilder`?

If the class gets large, we may as well split it into multiple ones (`WaitStepGenerator`/`WaitStepBuilder`) so that could maybe work together...